openmetadata-ui/src/main/resources/ui/cypress/common/SoftDeleteFlowUtils.js

```diff
@@ -188,7 +188,11 @@ export const softDeletedEntityCommonChecks = ({
 
   cy.get('[data-testid="manage-button"]').click();
   cy.get('[data-testid="delete-button-title"]').click();
-  cy.get('[data-testid="confirmation-text-input"]').type('DELETE');
+  cy.get('[data-testid="confirm-button"]').should('be.disabled');
+  cy.get('[data-testid="confirmation-text-input"]').type('DEL{enter}');
+  cy.get('#deleteTextInput_help').contains('Please type DELETE to confirm.');
+  cy.get('[data-testid="confirm-button"]').should('be.disabled');
+  cy.get('[data-testid="confirmation-text-input"]').clear().type('DELETE');
```

Comment:
Can we add an assertion to check button should not be disabled once the user has typed the `DELETE` text?

Reply:
@Sachin-chaurasiya, it will be an unnecessary assertion like should('be.visible') since we are anyway clicking on the button in the next step. If it is disabled, that action will fail.